packages/hydrooj/src/model/problem.ts

```diff
@@ -467,14 +467,15 @@ export class ProblemModel {
         try {
             const problems = await fs.readdir(tmpdir, { withFileTypes: true });
             for (const p of problems) {
+                if (process.env.HYDRO_CLI) logger.info(`Importing problem ${p.name}`);
                 const i = p.name;
                 if (!p.isDirectory()) continue;
                 const files = await fs.readdir(path.join(tmpdir, i));
                 if (!files.includes('problem.yaml')) continue;
                 const content = fs.readFileSync(path.join(tmpdir, i, 'problem.yaml'), 'utf-8');
                 const pdoc: ProblemDoc = yaml.load(content) as any;
                 if (!pdoc) continue;
-                let pid = pdoc.pid;
+                let pid: string | undefined = pdoc.pid;
```

Comment:
`strictNullCheck` is disabled globally in `tsconfig.json`. Adding this annotation actually does nothing.